datadog_operator/metadata.csv

```diff
@@ -1,2 +1,69 @@
+datadog.operator.go_memstats_other_sys_bytes,gauge,,byte,,Number of bytes used for other system allocations,0,datadog_operator,go_memstats_other_sys_bytes,
+datadog.operator.go_memstats_stack_inuse_bytes,gauge,,byte,,Number of bytes in use by the stack allocator,0,datadog_operator,go_memstats_stack_inuse_bytes,
+datadog.operator.go_memstats_stack_sys_bytes,gauge,,byte,,Number of bytes obtained from system for stack allocator,0,datadog_operator,go_memstats_stack_sys_bytes,
+datadog.operator.go_memstats_sys_bytes,gauge,,byte,,Number of bytes obtained from system,0,datadog_operator,go_memstats_stack_sys_bytes,
+datadog.operator.go_threads,gauge,,thread,,Number of OS threads created,0,datadog_operator,go_threads,
+datadog.operator.ksm.feature.enabled,gauge,,,,`1` if the Kubernetes State Metrics Core feature is enabled,0,datadog_operator,ksm_enabled,
+datadog.operator.live_container.feature.enabled,gauge,,,,`1` if the Live Containers feature is enabled,0,datadog_operator,live_containers_enabled,
+datadog.operator.log_collection.feature.enabled,gauge,,,,`1` if the Log Collection feature is enabled,0,datadog_operator,log_collection_enabled,
+datadog.operator.orchestrator_explorer.feature.enabled,gauge,,,,`1` if the Kubernetes Orchestrator Explorer feature is enabled,0,datadog_operator,orchestrator_explorer_enabled,
+datadog.operator.process_cpu_seconds_total,count,,,,Total user and system CPU time spent in seconds,0,datadog_operator,process_cpu_seconds_total,
+datadog.operator.process_max_fds,gauge,,,,Maximum number of open file descriptors,0,datadog_operator,process_max_fds,
+datadog.operator.process_open_fds,gauge,,,,Number of open file descriptors,0,datadog_operator,process_open_fds,
+datadog.operator.process_resident_memory_bytes,gauge,,byte,,Resident memory size in bytes,0,datadog_operator,process_resident_memory_bytes,
+datadog.operator.process_start_time_seconds,gauge,,second,,Start time of the process since unix epoch in seconds,0,datadog_operator,process_start_time_seconds,
+datadog.operator.process_virtual_memory_bytes,gauge,,byte,,Virtual memory size in bytes,0,datadog_operator,process_virtual_memory_bytes,
+datadog.operator.process_virtual_memory_max_bytes,gauge,,byte,,Maximum amount of virtual memory available in bytes,0,datadog_operator,process_virtual_memory_max_bytes,
+datadog.operator.reconcile.success,gauge,,,,"`1` if the last recorded reconcile error is null, `0` otherwise. The `reconcile_err` tag describes the last recorded error.",0,datadog_operator,reconcile_success,
+datadog.operator.remote_config.feature.enabled,gauge,,,,`1` if the Remote Configuration feature is enabled,0,datadog_operator,remote_config_enabled,
+datadog.operator.rest_client_requests_total,count,,,,"Number of HTTP requests, partitioned by status code, method, and host",0,datadog_operator,rest_client_requests_total,
```

Comment:
we have a `request` unit

Reply:
Addressed by https://github.com/DataDog/integrations-core/pull/15655/commits/bd9e3c25d67d1eeae6641e025d62d19c94c77e27